paimon-presto-common/src/main/java/org/apache/paimon/presto/PrestoTableHandle.java

```diff
@@ -33,14 +35,20 @@
 
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 /** Presto {@link ConnectorTableHandle}. */
 public class PrestoTableHandle implements ConnectorTableHandle {
 
+    public static final String SCAN_TIMESTAMP = "scan_timestamp_millis";
+    public static final String SCAN_DATETIME = "scan_datetime";
```

Comment:
We should remove this option, if it is required, it should be done in paimon-core.